src/neptune/objects/run.py

```diff
@@ -104,6 +106,21 @@
     from neptune.internal.background_job import BackgroundJob
 
 
+T = TypeVar("T")
+
+
+def temporarily_disabled(func: Callable[..., T]) -> Callable[..., T]:
+    def wrapper(*_, **__):
+        if func.__name__ == "_get_background_jobs":
+            return []
+        elif func.__name__ == "_write_initial_attributes":
+            return None
+        elif func.__name__ == "_write_initial_monitoring_attributes":
```

Comment:
I kinda don't like an idea of entangling `temporarily_disabled` with names of function from `Run`.
We could  add extra param `temporarily_disabled` to that defines return value so `Run` method signature does not change. But if its temporary it doesnt matter I guess